src/presenter/board-presenter.js

```diff
@@ -13,10 +13,45 @@ export default class BoardPresenter {
     this.boardPoints = [...this.pointsModel.getPoints()];
 
     render(this.boardComponent, this.boardContainer);
-    render(new PointEditView(this.boardPoints[0], this.pointsOffers), this.boardComponent.getElement());
+    // render(new PointEditView(this.boardPoints[0], this.pointsOffers), this.boardComponent.getElement());
 
-    for (let i = 0; i < this.boardPoints.length; i++) {
-      render(new PointView(this.boardPoints[i], this.pointsOffers), this.boardComponent.getElement());
-    }
+    // for (let i = 0; i < this.boardPoints.length; i++) {
+    //   render(new PointView(this.boardPoints[i], this.pointsOffers), this.boardComponent.getElement());
+    // }
+  };
+
+  #renderPoint = (point) => {
+    const pointComponent = new PointView(point);
+
+    const pointEditComponent = new PointEditView(point);
+
+    const replaceCardToForm = () => {
+      this.boardComponent.element.replaceChild(pointEditComponent.element, pointComponent.element);
+    };
+
+    const replaceFormToCard = () => {
+      this.boardComponent.element.replaceChild(pointComponent.element, pointEditComponent.element);
+    };
+
+    const onEscKeyDown = (evt) => {
+      if (evt.key === 'Escape' || evt.key === 'Esc') {
+        evt.preventDefault();
+        replaceFormToCard();
+        document.removeEventListener('keydown', onEscKeyDown);
+      }
+    };
+
+    pointComponent.element.querySelector('.event__rollup-btn').addEventListener('click', () => {
+      replaceCardToForm();
+      document.addEventListener('keydown', onEscKeyDown);
+    });
+
+    pointEditComponent.element.querySelector('form').addEventListener('submit', (evt) => {
```

Comment:
Так же, как и в случае с  `.querySelector('.event__rollup-btn')` лучше вынести этот селектор, но уже во вьюшку point-edit-view, сделать например метод addFormSubmitListener(listener) по аналогии вместо этого
```suggestion
    pointEditComponent.addFormSubmitListener((evt) => {
```